src/lwc/gsResources/gsResources.js-meta.xml

```diff
@@ -2,7 +2,7 @@
 <LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
     <apiVersion>49.0</apiVersion>
     <isExposed>true</isExposed>
-    <masterLabel>NPSP Getting Started Resources</masterLabel>
+    <masterLabel>NPSP Get Started Page Resources</masterLabel>
```

Comment:
We might as well bring all the API versions up to v50 along with these changes. I've been noting that they need to be consistent across the project.